app/services/update_user.rb

```diff
@@ -40,9 +41,24 @@ def phone_attributes
       phone: attributes[:phone],
       confirmed_at: attributes[:phone_confirmed_at],
       delivery_preference: attribute(:otp_delivery_preference),
+      made_default_at: made_default_at_date,
     }.delete_if { |_, value| value.nil? }
   end
 
+  def made_default_at_date
+    return present_made_default_at if attributes[:otp_make_default_number].blank?
+    Time.zone.now
+  end
+
+  def present_made_default_at
```

Comment:
A bit of a nit here, but what if we renamed this `current_made_default_at`. `present_made_default_at` makes me think we're trying to test the presence of `made_default_at`